ansible/roles/vendor/gsa.ansible-role-beats/molecule/default/create.yml

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
Usually you don't need to include any molecule playbooks except for `playbook.yml`, molecule has some default create, destroy, etc playbooks baked in that it will use.

Are these modified from the default?